cryptic-game/frontend
# Updated Editor

**Description**
Added an "Edit" Feature to the Texteditor.
Added a deleted File Popup and changed File Popup to the Texteditor.

**Issue**
Closes #219 

**Testing Instructions**
- Open a file and Write something.
- Open a new Editor an open the same File.
- Save the Changes of the first Editor
- In the 2nd Editor should be a Popup.
- Now remove the File.
- Now should be an other Popup in both Editors

**Important**
You can destroy Wallets by Edit them!
